include/bag.h

```diff
@@ -23,101 +23,101 @@ typedef enum RegisterItemResult {
  * buttons. This is saved to flash.
  */
 typedef struct BagData {
-    ITEM_SLOT items[NUM_BAG_ITEMS];                // General items
-    ITEM_SLOT keyItems[NUM_BAG_KEY_ITEMS];         // Key items
-    ITEM_SLOT TMsHMs[NUM_BAG_TMS_HMS];             // Move machines
-    ITEM_SLOT mail[NUM_BAG_MAIL];                  // Mail items
-    ITEM_SLOT medicine[NUM_BAG_MEDICINE];          // Healing items
-    ITEM_SLOT berries[NUM_BAG_BERRIES];            // Berries
-    ITEM_SLOT balls[NUM_BAG_BALLS];                // Balls
-    ITEM_SLOT battleItems[NUM_BAG_BATTLE_ITEMS];   // Battle-only items
+    BagPocket items[NUM_BAG_ITEMS];                // General items
+    BagPocket keyItems[NUM_BAG_KEY_ITEMS];         // Key items
+    BagPocket TMsHMs[NUM_BAG_TMS_HMS];             // Move machines
+    BagPocket mail[NUM_BAG_MAIL];                  // Mail items
+    BagPocket medicine[NUM_BAG_MEDICINE];          // Healing items
+    BagPocket berries[NUM_BAG_BERRIES];            // Berries
+    BagPocket balls[NUM_BAG_BALLS];                // Balls
+    BagPocket battleItems[NUM_BAG_BATTLE_ITEMS];   // Battle-only items
     u16 registeredItems[2];                        // IDs of registered key items
-} BAG_DATA;
+} Bag;
```

Comment:
this should match with the name after typedef struct, either rename that to Bag, or rename this to BagData